services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/NodeManagerImpl.java

```diff
@@ -313,12 +308,12 @@ public Node getNodeForVersionNumber(UserInfo userInfo, String nodeId, Long versi
 	public Node update(UserInfo userInfo, Node updated)
 			throws ConflictingUpdateException, NotFoundException,
 			DatastoreException, UnauthorizedException, InvalidModelException {
-		return update(userInfo, updated, null, false);
+		return update(userInfo, updated, null, null, false);
 	}
 
 	@WriteTransaction
 	@Override
-	public Node update(UserInfo userInfo, Node updatedNode, NamedAnnotations updatedAnnos, boolean newVersion)
+	public Node update(UserInfo userInfo, Node updatedNode, Annotations entityPropertyAnnotations, Annotations userAnnotations, boolean newVersion)
```

Comment:
check usage to see if userannotations is even necessary as  a parameter

Reply:
also no need to validate entity property annotations since it's only used between entitymanager and nodemanager and generated by a translation utility